crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -4338,8 +4403,15 @@ mod tests {
         let class = ty.expect_class_literal();
 
         let base_names: Vec<_> = class
-            .bases(&db)
-            .map(|base_ty| format!("{}", base_ty.display(&db)))
+            .node(&db)
```

Comment:
TBH I feel you could just delete this test. It doesn't test anything that isn't covered much better by your added MRO tests in this PR.